common/metrics/metric_defs.go

```diff
@@ -1222,6 +1222,7 @@ var (
 	ServiceFailures                               = NewCounterDef("service_errors")
 	ServiceErrorWithType                          = NewCounterDef("service_error_with_type")
 	ServiceCriticalFailures                       = NewCounterDef("service_errors_critical")
+	ServicePanic                                  = NewCounterDef("service_panics")
```

Comment:
I feel `service_request_panics` is better name to use. but leave `service_panics` here to follow the convention. lmkwdyt